### ISSUE TITLE
Brief doesnt retrieve articles from certain sites

### ISSUE THREAD

**Issue by fbrau** [brief-rss/brief#429]
Hi, I  migrated last week from Feedbro and Im satisfied by the simpler interface here. 

To get on track I exported my subscriptions from Feedbro and imported to Brief. All my subscriptions where migrated correctly but now for some sites Brief doesnt retrieve feed updates at all. 

These are the sites:

https://nautil.us/
http://lxer.com/
https://blog.mozilla.org/
https://www.djangoproject.com/weblog/

Whats wrong with these that  Brief seems to be incompatible?

**Comment by fbrau** [comment]
Update: Since I published the issue, the nautilus and mozilla feeds were updated by Brief with new articles. The others, lxer and django's, remain empty.

**Comment by jakobon** [comment]
Brief stopped to synhcronize all feeds two weeks ago, so today I followed the advise to remove Brief and add it again. Sorry to see that all my feeds was emptied. Beside this, about half of my feeds are empty, despite the website contains content.

The affected feeds include:
https://blogs.technet.microsoft.com/heyscriptingguy/
https://blogs.technet.microsoft.com/pki/
https://techcommunity.microsoft.com/t5/Enterprise-Mobility-Security/bg-p/enterprisemobilityandsecurity
https://www.microsoft.com/en-us/us-partner-blog/

Also, I cannot add new feeds, like this one:
https://www.jetglobal.com/blog/

When I press the "Subscribe to our blog" RSS feed link, I just get the following dialog window:
"Opening.
 You have chosen to open: <blank>
 which is:  application/rss+xml (xx.x KB)
from https://www.jetglobal.com
What should Firefox do with this file?
-> Open File [Browse]
-> Save file ..."

Basically Firefox doesn't know how to deal with the RSS feed file. This has been the case since the Mozilla team closed down support for Live Bookmarks, which by the way was a bad decision, at least in my view.

Any plans for releasing an update that can solve these issues ?
Besides this, thanks for a great feed reader !

**Comment by n8v8R** [comment]
For debugging purposes it is recommended to mention the OS (version, x_86/64), browser version  (x_86/64), extension version.

Except for jetglobal all the other feeds are working for me on a node with W10 v1809 b17763.437, FF 68.0a1 (2019-05-01) (64-bit) and Brief 2.59.

As for jetglobal this might the same issue as https://github.com/brief-rss/brief/issues/436

**Comment by tanriol** [comment]
For `jetglobal` it was #411. The other feeds seem to work for now.
